index.js

```diff
@@ -1,6 +1,43 @@
 module.exports = {
-    gltfPipeline : require('./lib/gltfPipeline'),
+    AccessorReader : require('./lib/AccessorReader'),
```

Comment:
I think manually maintaining this list could be error prone and easy to forget about.  Some other options would be to have a `private` folder at the same level of `lib` to make the decision more conscious during development; or you could just export everything and mark private objects `@private`.

That being said, I'm okay if you go the manual route until it becomes a pain.